providers/README.md

```diff
@@ -1,10 +0,0 @@
-Providers are now shipped as plugins to enable users develop their own providers in a seperate repository.
-
-CloudQuery Hub: https://hub.cloudquery.io
-
-Official plugins are located:
-- https://github.com/cloudquery/cq-provider-aws
-- https://github.com/cloudquery/cq-provider-azure
-- https://github.com/cloudquery/cq-provider-gcp
-- https://github.com/cloudquery/cq-provider-okta
-- https://github.com/cloudquery/cq-provider-ok8s
```

Comment:
Why this file is deleted? I think some people go through the code and look for the providers so it's nice to have a stub with README here imo?

Reply:
Added it again, although eventually it should be removed